Add bump-version and tag-version scripts (#32)
mongodb-labs/drivers-github-tools: README.md

    command: "git tag -m 'Tag' -s --local-user=${{ env.GPG_KEY_ID }} -a <tag>"
```

### bump-version

This is a convenience action to bump the version, create a signed commit, and

> push the commit if it is not a dry run.  You can override the commit message

Reviewer: ```suggestion
push the commit unless `push_commit` is disabled.  You can override the commit message
```

format if desired.  The version bump script should accept a new version as
an argument and update the version accordingly.

```yaml
- name: Bump version
  uses: mongodb-labs/drivers-github-tools/bump-version@v2
  with:
    version: ${{ inputs.version }}
    version_bump_script: "bash ./my-bump-version-script.sh"

>     dry_run: ${{ inputs.dry_run }}

Reviewer: Suggest removing it since this is the default usage
```suggestion
```

```

### tag-version

This is a convenience action to create a signed tag, optionally verify the tag,

> and push the tag if it is not a dry run.  You can override the tag format and the

Reviewer: ```suggestion
and push the tag unless `push_tag` is disabled.  You can override the tag format and the
```

tag message format if desired.

```yaml
- name: Tag version
  uses: mongodb-labs/drivers-github-tools/tag-version@v2
  with:
    version: ${{ inputs.version }}

>     dry_run: ${{ inputs.dry_run }}

Reviewer: Suggest removing since this is the default usage
```suggestion
```
